src/pytorch_lightning/accelerators/hpu.py

```diff
@@ -39,9 +39,24 @@ def setup_environment(self, root_device: torch.device) -> None:
             raise MisconfigurationException(f"Device should be HPU, got {root_device} instead.")
 
     def get_device_stats(self, device: Union[str, torch.device]) -> Dict[str, Any]:
-        """HPU device stats aren't supported yet."""
-        rank_zero_debug("HPU device stats aren't supported yet.")
-        return {}
+        """Returns a map of the following metrics with their values:
+
+        Limit,
+        InUse,
+        MaxInUse,
+        NumAllocs,
+        NumFrees,
+        ActiveAllocs,
+        MaxAllocSize,
+        TotalSystemAllocs,
+        TotalSystemFrees,
+        TotalActiveAllocs
```

Comment:
can you check the alignment in the docs here?

Reply:
@rohitgr7 what do you mean here?

Reply:
updated